packages/merge-on-green/test/merge-on-green.ts

```diff
@@ -857,7 +851,7 @@ describe('merge-on-green', () => {
       ));
 
       await probot.receive({
-        name: 'pull_request.labeled',
+        name: 'pull_request.labeled' as any,
```

Comment:
I opened an issue related to this, and it seems like we should be able to work around the issue like this:

https://github.com/probot/probot/issues/1367#issuecomment-700416806

So I think hopefully the only actual issue is the `repository.scheduled`.